lesson23/js/task_8.js

```diff
@@ -0,0 +1,20 @@
+// Запросить у пользователя длину окружности и периметр квадрата.
+// Определить, может ли такая окружность поместиться в указанный квадрат.
+const lengthOfCircle = +prompt(
+  "Enter the length of the circle in centimeters:"
+);
+const perimeterOfSquare = +prompt(
+  "Enter the perimeter of the square in centimeters:"
+);
+const PI = 3.1415926535897932384626433832795;
+const radius = lengthOfCircle / (2 * PI);
```

Comment:
можна також взяти з **Math**
```suggestion
const radius = lengthOfCircle / (2 * Math.PI);
```